apple/RNCWebViewImpl.m

```diff
@@ -564,21 +564,9 @@ - (void)initializeWebView
   }
 }
 
-// react-native-mac os does not support didMoveToSuperView
-#if !TARGET_OS_OSX
-- (void)didMoveToSuperview
-{
-  if (self.superview != nil && _webView == nil) {
-    [self initializeWebView];
-  }
-}
-#endif // !TARGET_OS_OSX
-
 - (void)didMoveToWindow
 {
-  if (self.window != nil && _webView == nil) {
-    [self initializeWebView];
-  }
+  [self initializeWebView];
```

Comment:
I am not sure if this is needed or not with the call from `setSource:` below since from my testing that seems to be called first so here it doesn't have any effect. I thought it would be safer to leave it here too for now.